tests/test_TFNetworkRecLayer.py

```diff
@@ -5294,6 +5294,79 @@ def test_trafo_search_lm():
       assert out_seq_lens[i] == input_seq_lens[i] * 3 // 2  # we constructed the 'end' layer that way
       assert all(out_seqs[i, :input_seq_lens[i]] == input_seqs[i, :input_seq_lens[i]])
```

Comment:
Note that there should be 2 empty lines before a new function (always check PyCharm warnings).